storybook/helpers/ComponentView.tsx

```diff
@@ -55,26 +56,29 @@ export default class ComponentView extends React.Component<Props, State> {
           <Component.Entity>
           <Component.Data>
           <Component.Logs>
-          {(entity: any, data: any, logs: ComponentLogs) => (
-            <>
-            <Typography variant="h6" component="h6">
-              Entity
-            </Typography>
-            {objectInspector(entity, `${name}.Entity`, "Entity Instance")}
-            <Typography variant="h6" component="h6">
-              Data
-            </Typography>
-            {objectInspector(data, `${name}.Data`, "Semantic Data")}
-            <Typography variant="h6" component="h6">
-              Logs
-            </Typography>
-            {objectInspector(logs.react, `${name}.ReactLogs`, "React Logs")}
-            {objectInspector(logs.entity, `${name}.EntityLogs`, "Entity Logs")}
-            {objectInspector(logs.data, `${name}.DataLogs`, "Data Query Logs")}
-            {objectInspector(logs.code, `${name}.CodeLogs`, "Code Logs")}
-            {objectInspector(logs.prose, `${name}.ProseLogs`, "Prose Logs")}
-            </>
-          )}
+          {(entity: any, data: any, logs: ComponentLogs) => {
+            if ( !entity || !data)
```

Comment:
My other comment says this, but let's not handle the loading here, as this is an example user view. Let's do it within the ContextFeed.